modules/ROOT/pages/create-or-modify-a-page.adoc

```diff
@@ -1,65 +1,178 @@
+====
+
+For more technical information about the pages, visit xref:pages-development.adoc[the dedicated documentation page]. 
+
+== Create a layout
+
+=== From the provided Bonita layout
+==== If you have not installed any Bonita application in your project in Bonita Studio 
+
+. From Bonita Studio, click on the *_Applications_* icon and select either the Administrator or Super Administrator application
+. Go to the *_Resources_* menu option
+. If needed, filter the type by _Layout_, and select *_Bonita Layout_* 
+. Click on the *_Export_* button
+. From Bonita Studio, click on the *_UI Designer_* icon to open the UI Designer
+. Click on the *_Import_* button 
+. Find the Bonita layout you have exported and load it
+. Edit it and save it with another name
+
+==== If you have installed a Bonita application 
+
+. In Bonita Studio, go to the _Project Explorer_
+. Open the project, and the _Pages/Forms/Layouts_ section
+. Douvle click on _Bonita layout_. This automatically opens the UI Designer with the layout ready to edit.
+. Edit it and save it with another name
+
+=== From scratch
+
+Use the same sequence of steps than to create a page from scratch.
+
+For more technical information about layouts, visit the xref:layout-development.adoc[dedicated documentation page].
+
+== Create a fragment
+
+Use the same sequence of steps than to create a page from scratch.
+
+
+== Responsiveness
 
 You can optimize the page or form design for a given device type by configuring xref:widget-properties.adoc[device-specific values for the Width property] for widgets. Use the a device types bar in the Page editor to choose the target device type.
 image:images/images-6_0/pb-resolution.png[Device type selection]
 
-Custom widget is a special artifact that we will explain in a later chapter.
+== Modify an element
 
-== Preview a page, layout, form, or fragment
+After a form, page, layout, or fragment is created, you can update it by just clicking on the *_Pencil_* icon to open it in the editor. +
+Update the element by deleting or dragging and dropping widgets, business objects or fragments from the palette to the whiteboard. +
+Edit widget properties in the widget properties panel, create data, and bind them to the widgets.
 
-You can preview page, form and fragment rendering by clicking the *_Preview_* button. Another browser window opens with the artifact as it will be displayed after deployment. If you update the page, form or fragment, the preview is automatically refreshed when you save.
+== Preview an element
 
-You can display your page, layout, form or fragment with one of your theme application installed.
+You can preview page, form and fragment rendering by clicking the *_Preview_* button. Another browser window opens with the element as it will be displayed after deployment. If you update the page, form or fragment, the preview is automatically refreshed when you save. +
```

Comment:
```suggestion
You can preview page, form and fragment rendering by clicking on the *_Preview_* button. Another browser window opens with the element as it will be displayed after deployment. If you update the page, form or fragment, the preview is automatically refreshed when you save. +
```